src/platform/Linux/ConnectivityManagerImpl.cpp

```diff
@@ -1061,6 +1061,24 @@ void ConnectivityManagerImpl::_ConnectWiFiNetworkAsyncCallback(GObject * source_
                 mpConnectCallback = nullptr;
             });
         }
+        else
```

Comment:
nit: adding a return in the if above would allow us to decrease indent. I think that is prefferable to adding more indent on else on every success path.

Reply:
acked